src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -30,16 +30,14 @@
  * Checks that all package annotations are in the package-info.java file.
  * </p>
  * <p>
- * According to the Java Language Specification.
+ * For Java SE8 and above, placement of package annotations in the package-info.java
+ * file is enforced by the compiler and this check is not necessary. <br/>
```

Comment:
Cool, one last nitpick, wonder about a new p here instead of a br

Reply:
Done.